planck-cljs/src/planck/console.cljs

```diff
@@ -0,0 +1,59 @@
+(ns ^:no-doc planck.console
+  "Implementation of js/console"
+  (:refer-clojure :exclude [time]))
+
+;; Base logging functions
+
+(defn- log-stdout [& args]
+  (apply js/PLANCK_CONSOLE_STDOUT args))
+
+(defn- log-stderr [& args]
+  (apply js/PLANCK_CONSOLE_STDERR args))
+
+;; Timing functions
+;; See https://console.spec.whatwg.org/#timing
+
+(def ^:private timer-state (atom {}))
+
+(defn- add-label [state label start-time]
+  (-> state
+    (assoc :label-exists? (contains? (:timer-table state) label))
+    (assoc-in [:timer-table label] start-time)))
+
```

Comment:
It looks to me in [the spec](https://console.spec.whatwg.org/#time), that if the label exists, the time shouldn't be updated but that's not what happens here, right? The `assoc-in` function will set the value regardless of whether the label exists or not.

Reply:
Good catch. Will fix.

Reply:
1 out of 5 ain't bad, I guess :P